### DISCUSSION TITLE
I downloaded the model to another file path, how do I write the command？

### DISCUSSION THREAD

**Discussion by geng-lee** [brianhie/efficient-evolution#32] [Q&A]
Hi，Developer！

I downloaded the model to another file path, how do I write the command？

python bin/recommend.py $wildtype_sequence ？

Best,
Jamie

**Comment by brianhie** [comment]
Hi @geng-lee, if you do `pip install fair-esm`, the corresponding package (`esm`) should automatically download the model for you (by default, into `~/.cache/torch`). You can then run the `recommend.py` command from anywhere. Hope that helps.

**Reply by geng-lee** [reply]
I know, but this is very inconvenient, so I copied the download link and downloaded the models into a custom folder. I specified it with this command, but it didn't work. I want to know how to add a custom model call path to the command.

--model-names /storage/Lab/soft/efficient-evolution/models/esm1b_t33_650M_UR50S.pt

**Reply by brianhie** [reply]
See here for how to change the place where torch hub stores models: https://stackoverflow.com/questions/59134499/how-to-modify-path-where-torch-hub-models-are-downloaded

**Reply by geng-lee** [reply]
😁 Ok, thank you very much! I'll try.

**Reply by geng-lee** [reply]
Hi, @brianhie

Can efficient-evolution store all mutation results in csv files, and then how to obtain mutation probabilities or score matrices and visualize them?
